tests/data/invalid/app/no_lang_version.json

```diff
@@ -1,10 +0,0 @@
-{
-    "name": "app1",
-    "agent": {
-      "name": "python",
-      "version": "1.0"
```

Comment:
maybe also remove the version from the valid json, to make sure it works without it, but if you tested it is fine